website/common/locales/en/newClient.json

```diff
@@ -98,7 +98,6 @@
   "yourNotOnQuest": "You're not on a quest",
   "questDescription": "Quests allow players to focus on long-term, in-game goals with the members of their party.",
   "haveNoChallenges": "You don’t have any Challenges",
-  "challengeDescription": "Challenges are community events in which players compete and earn prizes by completing a group of related tasks.",
```

Comment:
I've deleted this because `challengeDescription` was used elsewhere for a different string, and that string there is duplicated in `challengeDescription0`.